docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -408,8 +408,8 @@ The following table provides an overview of how different Next.js APIs affect ca
 | [`fetch`](#fetch)                                                      |              |                       | Cache                 | Cache       |
 | [`fetch` `options.cache`](#fetch-optionscache)                         |              |                       | Cache or Opt out      |             |
 | [`fetch` `options.next.revalidate`](#fetch-optionsnextrevalidate)      |              | Revalidate            | Revalidate            |             |
-| [`fetch` `options.next.tags`](#fetch-optionsnexttag-and-revalidatetag) |              | Cache                 | Cache                 |             |
-| [`revalidateTag`](#fetch-optionsnexttag-and-revalidatetag)             |              | Revalidate            | Revalidate            |             |
+| [`fetch` `options.next.tags`](#fetch-optionsnexttags-and-revalidatetag) |              | Cache                 | Cache                 |             |
```

Comment:
Could we run prettier here please?

Reply:
Wait, I can't tell from the diff if it's needed - approving and if it passes, it will merge!

Reply:
ran prettier and pushed it :)